webpack.config.dev.js

```diff
@@ -52,19 +55,26 @@ const devConfig = {
       title: 'Polymesh SDK - Sandbox',
     }),
     new CaseSensitivePathsPlugin(),
+    // Work around for Buffer is undefined:
+    // https://github.com/webpack/changelog-v5/issues/10
+    new webpack.ProvidePlugin({
+      Buffer: ['buffer', 'Buffer'],
+    }),
   ],
   output: {
     pathinfo: true,
     filename: 'devServer.js',
     path: path.resolve(__dirname, 'dist'),
     publicPath: '/',
   },
-  node: {
-    fs: 'empty',
-  },
   resolve: {
     extensions: ['.ts', '.js'],
     plugins: [new TsconfigPathsPlugin()],
+    fallback: {
+      crypto: require.resolve('crypto-browserify'),
```

Comment:
We need to explicitly add in these polyfills, the old version would automatically include them which would inflate the bundle size.